Modules/Setup.stdlib.in

```diff
@@ -113,6 +113,7 @@
 @MODULE_NIS_TRUE@nis nismodule.c
 # needs sys/soundcard.h or linux/soundcard.h (Linux, FreeBSD)
 @MODULE_OSSAUDIODEV_TRUE@ossaudiodev ossaudiodev.c
+@MODULE__POSIXSHMEM_TRUE@_posixshmem _multiprocessing/posixshmem.c
```

Comment:
... or we could move this to a "multiprocessing" section

Reply:
Yeah. that makes more sense.